rules-reviewed/fuse/sonicesb/xml-sonic-esb.windup.xml

```diff
@@ -22,7 +22,9 @@
                 </xmlfile>
             </when>
             <perform>
-                <classification title="SonicESB Service Configuration" effort="4"/>
+                <classification title="SonicESB Service Configuration" effort="4">
+                    <tag>SonicESB</tag>
```

Comment:
well, if we do prefer all lowercase I will change all others too like "Apache Camel" to "camel" WDYT?